tasks/analysis.rake

```diff
@@ -6,32 +6,31 @@ namespace :analysis do
       
       # Roll through events in date order and create incidents representing consecutive series of down events
       incident = nil
-      DB["monitor_events"].find({"monitor" => monitor["id"]}).sort({date: 1}).each do |event|
-        if event["status"] == 0
+      MonitorEvent.where(monitor: monitor['id']).sort({date: 1}).each do |event|
+        if event.status == 0
```

Comment:
an alternative is to let the model only do persistence and we create a decorator that has this logic.